plugins/jfrog-artifactory/src/components/useJfrogArtifactoryAppData.ts

```diff
@@ -1,15 +1,16 @@
 import { Entity } from '@backstage/catalog-model';
 
-export const JFROG_ARTIFACTORY_ANNOTATION_IMAGE_NAME =
-  'jfrog-artifactory/image-name';
+export const JFROG_ARTIFACTORY_ANNOTATION_ARTIFACT_NAME =
+  'jfrog-artifactory/artifact-name';
 
 export const useJfrogArtifactoryAppData = ({ entity }: { entity: Entity }) => {
-  const imageName =
-    entity?.metadata.annotations?.[JFROG_ARTIFACTORY_ANNOTATION_IMAGE_NAME] ??
-    '';
+  const artifactName =
+    entity?.metadata.annotations?.[
+      JFROG_ARTIFACTORY_ANNOTATION_ARTIFACT_NAME
```

Comment:
I'm thinking actually of changing this to `PACKAGE_NAME` instead of `ARTIFACT_NAME` 🤔